src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -16,6 +18,98 @@ def filter_notification_link(link_text, amount, filter_item)
     end
   end
 
+  def notification_icon(notifiable_type)
+    return image_tag('request_icon', size: 18, title: 'Request notification') if notifiable_type == 'BsRequest'
+
+    capture do
+      tag.i(class: 'fas fa-comments mr-1', title: 'Comment notification')
+    end
+  end
+
+  def request_state(notification)
+    return unless notification.notifiable_type == 'BsRequest'
+
+    state = notification.notifiable.state
+    color = request_badge_color(state)
+
+    capture do
+      tag.span(state, class: "badge badge-#{color}")
+    end
+  end
+
+  def request_badge_color(state)
+    case state
+    when :review, :new
+      'secondary'
+    when :declined, :revoke
+      'danger'
+    when :superseded
+      'warning'
+    when :accepted
+      'success'
+    else
+      'dark'
+    end
+  end
+
+  def notification_link(notification)
+    link_text = case notification.event_type
+                # TODO: add 'Event::ReviewWanted' when converted to BsRequest notification
+                when 'Event::RequestStatechange', 'Event::RequestCreate'
+                  "##{notification.notifiable.number}"
+                when 'Event::CommentForRequest'
+                  "Request ##{notification.notifiable.commentable.number}"
```

Comment:
Either `#7` or `Request #7`. Not both.

![Screenshot from 2020-07-08 14-11-53](https://user-images.githubusercontent.com/514785/86917214-00d2ff00-c125-11ea-97fe-49d031e2dbc3.png)

Reply:
`Request #7`then.